lib/qbwc/job.rb

```diff
@@ -83,7 +96,7 @@ def next
 
   def reset
     self.request_index = 0
-    self.requests = []
+    #self.requests = []
```

Comment:
With this change, when using requests generated by the worker, are the requests regenerated at the start of the job, and only at the start of the job?